ddev/src/ddev/repo/core.py

```diff
@@ -18,3 +27,38 @@ def name(self) -> str:
     @property
     def path(self) -> Path:
         return self.__path
+
+    @property
+    def integrations(self) -> IntegrationRegistry:
+        return self.__integrations
+
+    @cached_property
+    def config(self) -> RepositoryConfig:
+        from ddev.repo.config import RepositoryConfig
```

Comment:
Is this supposed to be for performance? I think this sort of dynamic import should be saved for very well-justified cases, I'm not sure that would be the case here.

Reply:
Yes part of the RFC was making imports lazy to reduce startup time

Reply:
Without measuring or profiling I'm a bit skeptical of this as something to be applied too generally.

Reply:
I'm absolutely certain that allowing >1ms imports over time adds up to a slow app https://github.com/pypa/hatch/actions/runs/3261755280/jobs/5357360561#step:7:1